tests/testresources/metrics/config/api.yml

```diff
@@ -9,47 +9,47 @@ gcp:
   featureSet: default
   dimensions:
   - value: resource.labels.project_id
-    id: project_id
+    key: project_id
     compatibilityModeOnly: true
   - value: resource.labels.project_id
-    id: gcp.project.id
+    key: gcp.project.id
   - value: resource.labels.service
-    id: service
+    key: service
   - value: resource.labels.method
-    id: method
+    key: method
   - value: resource.labels.version
-    id: version
+    key: version
   - value: resource.labels.location
-    id: location
+    key: location
     compatibilityModeOnly: true
   - value: resource.labels.location
-    id: gcp.region
+    key: gcp.region
   metrics:
   - value: serviceruntime.googleapis.com/api/request_count
-    id: cloud.gcp.serviceruntime_googleapis_com.api.request_count
+    key: cloud.gcp.serviceruntime_googleapis_com.api.request_count
     type: count,delta
     name: Request count
-    unit: '1'
     gcpOptions:
-      ingestDelay: 180s
-      samplePeriod: 60s
+      ingestDelay: 180
+      samplePeriod: 60
       valueType: INT64
       metricKind: DELTA
+      unit: '1'
     dimensions:
     - value: metric.labels.protocol
     - value: metric.labels.response_code
     - value: metric.labels.response_code_class
     - value: metric.labels.grpc_status_code
   - value: serviceruntime.googleapis.com/api/request_latencies
```

Comment:
can we have prefixes 'metric:' and 'label:' added in some of test configs?
I know they are optional, but it is worth to have it tested.